case-import-emulator/update_cgmes.sh

```diff
@@ -49,17 +76,44 @@ else
     cd $caseName
   fi
 
+  for xmlFile in $(ls *.xml); do
+    # memorizing old and new uuid, and setting new uuid for SV, TP, EQ, SSH profiles
+    if [[ "$xmlFile" =~ ^.*_(SV)_[0-9]+[.]xml$ ]]
+    then
+      oldUuidSV=$(grep 'md:FullModel.*rdf:about=' $xmlFile | cut -d'"' -f 2 | cut -d':' -f 3)
+      newUuidSV=$(uuidgen)
+      sed -i -e "s/\(<md:FullModel[ ]*rdf:about=\"urn:uuid:\).*\(\">\)/\1${newUuidSV}\2/g" $xmlFile
```

Comment:
Can't newly created "updateUuid()" method be used here (and for later identical calls) ?

Reply:
Done